web/test/pages/roadmap.test.tsx

```diff
@@ -175,247 +174,65 @@ describe("roadmap page", () => {
     expect(screen.getByText(Config.profileDefaults.successTextHeader)).toBeInTheDocument();
   });
 
-  it("directs guest-mode user to profile when profile edit button is clicked", () => {
+  it("shows date based on New Jersey Local time", () => {
+    const date = getCurrentDateInNewJersey().format("MMMM DD YYYY");
     useMockProfileData({});
-    const setModalIsVisible = jest.fn();
-    render(
-      withAuthAlert(
-        <ThemeProvider theme={createTheme()}>
-          <RoadmapPage operateReferences={{}} displayContent={createDisplayContent()} />
-        </ThemeProvider>,
-        IsAuthenticated.FALSE,
-        { modalIsVisible: false, setModalIsVisible }
-      )
-    );
-    fireEvent.click(screen.getByTestId("grey-callout-link"));
-    expect(mockPush).toHaveBeenCalled();
-    expect(setModalIsVisible).not.toHaveBeenCalled();
-  });
+    renderRoadmapPage({});
```

Comment:
this comment applies to most of the changes in this file - I think this would be a great opportunity to do a component test for <Header/> instead of having all these tests in the roadmap page

Reply:
will do